vm/actor_interface/src/builtin/miner/mod.rs

```diff
@@ -161,6 +186,23 @@ impl State {
                     Ok(infos)
                 }
             }
+            State::V10(st) => {
+                if let Some(sectors) = sectors {
+                    Ok(st
+                        .load_sector_infos(&store, sectors)?
+                        .into_iter()
+                        .map(From::from)
+                        .collect())
+                } else {
+                    let sectors = fil_actor_miner_v10::Sectors::load(&store, &st.sectors)?;
+                    let mut infos = Vec::with_capacity(sectors.amt.count() as usize);
+                    sectors.amt.for_each(|_, info| {
```

Comment:
This could be a 1-liner. sth like, amt.into_iter().map().collect()

Reply:
Unfortunately it cannot, as `sectors.amt` does not have `into_iter` or `iter` methods